packages/decoder/lib/types.ts

```diff
@@ -260,9 +260,9 @@ export interface Transaction {
    */
   gasPrice?: string;
   /**
-   * Gas provided by the sender, as numeric string.
+   * Gas provided by the sender, as number.
    */
-  gas?: string;
+  gas?: number;
```

Comment:
Hm, so, I just checked what web3 actually returns, and it's a string, not a number.  As such I'm wary of changing this type -- the reason that this type is defined here, after all, rather than just using web3's types, is because web3's types are so frequently wrong!

So rather than changing this type, could there be another solution?  Like, maybe we just don't use the type from `web3-core` in the first place?  (What if we import from `@types/web3`?  Or from `web3` directly, which I assume would take from `@types/web3`?)  Or could one just coerce if really necessary?  (Can the `web3-core` dependency just go away, perhaps...?)